superdesk/sams/assets.py

```diff
@@ -204,3 +204,16 @@ def unlock_asset_by_user(user_id, session_id):
             "sams:asset:session_unlock", user_id=get_user_id(True), session_id=get_auth()["_id"], extension="sams"
         )
     return unlock_asset_response.status_code
+
+
+@assets_bp.route("/sams/assets/tags/<search_query>", methods=["GET"])
```

Comment:
`search_query` should be a parameter not a suffix to the resource location.
In this case:

**resource**: /sams/assets/tags
**params**: query

This is because the resource here is `tags`, and `query` controls the response from that resource.

Example:
`GET /sams/assets/tags?query=Sports`

The `query` param should also be optional, so it should be possible to retrieve all `tags` from all Assets